js/main.js

```diff
@@ -61,14 +61,14 @@ $artButton.addEventListener('click', changePage3);
 function changePage3(event) {
```

Comment:
This function name doesn't describe the action that is taking place. Which page are you changing to?